src/sparseml/integration_helper_functions.py

```diff
@@ -51,10 +51,19 @@ def resolve_integration(
         will attempt to infer it from the source_path.
     :return: The name of the integration to use for exporting the model.
     """
-    from sparseml.pytorch.image_classification.utils.helpers import (
-        is_image_classification_model,
-    )
-    from sparseml.transformers.utils.helpers import is_transformer_model
+    try:
+        from sparseml.pytorch.image_classification.utils.helpers import (
+            is_image_classification_model,
+        )
+    except ImportError:
+        # unable to import integration, always return False
+        is_image_classification_model = _null_is_model
```

Comment:
should this be _null_is_model() like on line 66?

Reply:
66 should be like this one, good catch